Dockerfile.rhel7

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
```

Comment:
Is this change upstreamed?
what is the failure if not updating to go-1.13? like -modcache flag is not supported?

Reply:
yes the issue is with -modcache in the clean

Reply:
I didn't know we have this file on the u/s I will take a look and open the PR there